process-document/src/test/kotlin/com/ritense/processdocument/service/impl/CamundaProcessJsonSchemaDocumentDeploymentServiceIntTest.kt

```diff
@@ -16,31 +16,37 @@
 
 package com.ritense.processdocument.service.impl
 
+import com.ritense.authorization.AuthorizationContext
+import com.ritense.authorization.annotation.RunWithoutAuthorization
 import com.ritense.processdocument.BaseIntegrationTest
 import com.ritense.processdocument.domain.impl.CamundaProcessDefinitionKey
 import com.ritense.processdocument.service.ProcessDocumentAssociationService
 import com.ritense.processdocument.service.ProcessDocumentDeploymentService
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.transaction.annotation.Transactional
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
 @Transactional
 internal class CamundaProcessJsonSchemaDocumentDeploymentServiceIntTest : BaseIntegrationTest() {
 
+    @Autowired
```

Comment:
You can put these in a `@Autowired constructor(...)`

Reply:
Never quite understood the advantage to doing it like that. What do we gain from doing this exactly?